examples_rclcpp/src/action_client/switcher.cpp

```diff
@@ -0,0 +1,135 @@
+#include "action_client/switcher.hpp"
+
+using namespace robotis;
+using namespace std::chrono_literals;
+
+Switcher::Switcher(const int32_t input_value)
+: Node("switch"), goal_done_(false)
```

Comment:
Add new line before set new variable

Reply:
Added new line before set new variable and push